src/front/glsl/lex_tests.rs

```diff
@@ -10,28 +12,49 @@ fn glsl_lex_simple() {
 
     let mut iter = tokens.iter();
```

Comment:
why do we need to collect them? can we just call `next` on the iterator, and then at the end assert that the next() is None (instead of checking the count here)?

Reply:
good idea, makes writing the tests a bit simpler. Fixing ...